pytest_xvfb.py

```diff
@@ -132,6 +133,12 @@ def pytest_addoption(parser):
 def pytest_configure(config):
     if config.getoption('--no-xvfb') or not xvfb_available():
         config.xvfb = None
+        if (sys.platform.startswith('linux')
+                and 'DISPLAY' in os.environ
+                    and not config.getoption('--no-xvfb')):
+                        print(
+                            'pytest-xvfb could not find Xvfb. '
+                            'You can install it to avoid windows from opening.')
```

Comment:
Huh? The example sure does contain multiple statements:

```python
# Add some extra indentation on the conditional continuation line.
if (this_is_one_thing
        and that_is_another_thing):
    do_something()
```

So yes - 8 spaces for continuing the conditional, 4 spaces (as usual) for the body.